src/components/Sidebar.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import "./Sidebar.css";
+import Home from "../components/icons/home";
+import Menu from "../components/icons/menu";
+import Down from "../components/icons/down";
+
+interface SidebarProps {
+  setTitle: (title: string) => void;
+}
+
+const Sidebar: React.FC<SidebarProps> = ({ setTitle }) => {
+  const [showDropdown, setShowDropdown] = useState<boolean>(false);
+
+  return (
+    <div className="sidebar">
+      <div className="menu-item" onClick={() => setTitle("Start")}>
+        <Home /> Start
+      </div>
+      <div className="menu-item" onClick={() => setShowDropdown(!showDropdown)}>
+                  <Menu />Machine Learning &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<Down/>
+       
+      </div>
+      {showDropdown && (
+        <div className="dropdown">
+          <div className="dropdown-item" onClick={() => setTitle("Example 1")}>
```

Comment:
As you progress over tasks, you will see that content that changes is not just title on content. Each of the clicked menu entries will have its own page with separate content. React provides way to navigate between pages by using React Router and that is what you need to use in this application. Here are some useful links:
https://www.geeksforgeeks.org/reactjs-router/
https://reactrouter.com/en/main/start/tutorial

Please integrate React Browser Router into application and use it.